pontoon/contributors/templates/contributors/settings.html

```diff
@@ -117,18 +117,18 @@ <h3>Notification subscriptions</h3>
     </section>
 
     <section>
-        <h3>Editor Settings</h3>
+        <h3>Editor settings</h3>
         <ul class="check-list">
-            {{ Checkbox.checkbox('Translate Toolkit Checks', class='quality-checks', attribute='quality_checks', is_enabled=user.profile.quality_checks, title='Run Translate Toolkit checks before submitting translations') }}
+            {{ Checkbox.checkbox('Translate toolkit checks', class='quality-checks', attribute='quality_checks', is_enabled=user.profile.quality_checks, title='Run Translate Toolkit checks before submitting translations') }}
```

Comment:
Note that Toolkit is also part of the name of the library and is spelled with a capital T.
http://docs.translatehouse.org/projects/translate-toolkit/en/latest/index.html